core/src/Kokkos_Macros.hpp

```diff
@@ -533,6 +533,13 @@ static constexpr bool kokkos_omp_on_host() { return false; }
 #define KOKKOS_ENABLE_CUDA_LDG_INTRINSIC
 #endif
 
+#if defined(KOKKOS_ENABLE_IMPL_CUDA_EMULATE_UNIFIED_MEMORY)
+#define KOKKOS_ENABLE_IMPL_CUDA_UNIFIED_MEMORY
+#endif
+#if defined(KOKKOS_ARCH_ARMV9_GRACE) && defined(KOKKOS_ARCH_HOPPER90)
```

Comment:
Do we really need both? Isn't checking for `KOKKOS_ARCH_HOPPER90` sufficient?

Reply:
I think H100 + x86_64 machines have the same `KOKKOS_ARCH_HOPPER90` flag set and I am not sure all have HMM enabled.